### ISSUE TITLE
Add drop-out option?

### ISSUE THREAD

**Issue by hanfeisun** [pender/chatbot-rnn#12]
I found the char-rnn repo  (https://github.com/karpathy/char-rnn) provides the dropout option. Would it be possible for this feature to be added?

**Comment by pender** [comment]
Not planning to add this feature -- my understanding is that dropout primarily helps with overfitting, which shouldn't be an issue given how much training data is available from Reddit.
